code/game/machinery/newscaster/newscaster_machine.dm

```diff
@@ -954,6 +956,6 @@
 	icon_state = "newscaster"
 	custom_materials = list(/datum/material/iron=14000, /datum/material/glass=8000)
 	result_path = /obj/machinery/newscaster
-	pixel_shift = -32
+	pixel_shift = 30
```

Comment:
Same as before, previous 32, now 30

Reply:
No. The previous is -32. We added directional support for this item, so the pixel shift is not the same.